src/main/java/io/netty/incubator/codec/http3/Http3FrameValidationUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 The Netty Project
```

Comment:
2021

Reply:
![Tobias-sobbing-in-the-shower](https://user-images.githubusercontent.com/820005/121380512-c73ce200-c8f9-11eb-9fdb-a4035ee7e395.gif)